canonical-web-and-design/charmhub.io
# Topics pages on the homepage

## Done
- Topics pages on the homepage

## How to QA
- Check the demo link
- Filter by different categories and check that the filtering is working as expected.
- Perform a search by text + filter categories as well.

## Issue / Card
Fixes https://github.com/canonical-web-and-design/marketplace-tribe/issues/2345
